### DISCUSSION TITLE
posts/2024-08-23-remove_mask/

### DISCUSSION THREAD

**Discussion by giscus** [SkyDynamic/NynaBlog#5] [Announcements]
# posts/2024-08-23-remove_mask/

https://blog.skydynamic.top/posts/2024-08-23-remove_mask/

**Comment by Siwuxie-x** [comment]
#### 好！谢谢分享。非常实用的内容。
~~虽然这个不叫马赛克哇，一大格一大格的那个是马赛克。~~

**Reply by SkyDynamic** [reply]
模糊遮罩嘛，咱习惯这么说了，能知道意思就好/

**Comment by AbroadSphere** [comment]
b站电脑客户端要怎么去除呢
